### ISSUE TITLE
NbRangePicker Firefox always invalid

### ISSUE THREAD

**Issue by jmpmscorp** [akveo/nebular#1281]
### Issue type

**I'm submitting a ...**  (check one with "x")

* [x ] bug report
* [ ] feature request

### Issue description

**Current behavior:**
NbRangePicker is invalid in Firefox browsers, always has ng-invalid class. I have tested it with Chrome, Opera and Xiaomi mobile browser and there works without error. Only Firefox fail.

Firefox example:
![firefox rangepicker error](https://user-images.githubusercontent.com/22971731/53928329-fd8bb700-4089-11e9-929b-38c60a745464.jpg)

Chrome example (Opera, Xiaomi, too):
![chrome rangepicker error](https://user-images.githubusercontent.com/22971731/53928399-43e11600-408a-11e9-9dc5-a6a389c26dbc.jpg)

### Other information:

**npm, node, OS, Browser**
```
Node; 8.15.0
npm: 6.4.1
OS: Windows 10
Browser: Firefox

```

**Angular, Nebular**
```
Angular: 7.2.8
Nebular: 3.4.0
```

**Comment by yggg** [comment]
Hi @jmpmscorp, thanks for reporting! I'm not able to reproduce the issue, could you please provide steps or add code example?
I've created a starter with a range picker and date-fns on [stackblitz](https://stackblitz.com/edit/nebular-1281?file=src%2Fapp%2Fapp.component.ts). Form control is valid in firefox and other browsers.

**Comment by algirdasc** [comment]
I've encountered same problem. The bug can be reproduced using NbMomentService. 
`NbMomentDateService.prototype.isValidDateString` returns false on date range.

**Comment by jmpmscorp** [comment]
Excuse me @yggg. 

I was on holiday time and I cannot ask you. I'll try to test again and I'll write you. Since another bug with date-fns was solved on 3.4.x release, I change rangepicker by datepicker with date-fns, so, I'll test on a new project.

Writing from memory, I think I was using RangePicker with NbMomentDateModule as @algirdasc suggest when I wrote post, but honestly, I can't remember.

Regards.
